plugins/modules/win_dns_client.ps1

```diff
@@ -341,6 +359,22 @@ Function Set-DnsClientAddress {
     }
 }
 
+Function Set-DnsClientSuffixSearchList {
+    Param(
+        [string] $suffix_search_list
```

Comment:
This will coerce the string array to a string, should this be `[string[]]` instead?